app/vmalert/vmalertutil/links.go

```diff
@@ -11,7 +11,7 @@ const prefix = "/vmalert/"
 
 // Prefix returns "/vmalert/" prefix if it is missing in the path.
 func Prefix(path string) string {
-	pp := httpserver.GetPathPrefix()
+	pp := strings.TrimRight(httpserver.GetPathPrefix(), "/")
```

Comment:
please verify it doesn't break https://github.com/VictoriaMetrics/VictoriaMetrics/issues/4349

Reply:
checked, it works